examples/tts/conf/fastpitch/fastpitch_22050.yaml

```diff
@@ -0,0 +1,286 @@
+# This config contains the default values for training an English 22.05kHz FastPitch model.
+# If you want to train a model on other dataset, you can change config values according to your dataset.
+# Most dataset-specific arguments are in the head of the config file, see below.
+
+name: FastPitch
+
+max_epochs: ???
+batch_size: 32
+weighted_sampling_steps_per_epoch: null
+
+n_speakers: ???
+speaker_path: null
+feature_stats_path: null
+
+train_ds_meta: ???
+val_ds_meta: ???
+log_ds_meta: ???
+
+phoneme_dict_path: ???
+heteronyms_path: ???
+
+log_dir: ???
+vocoder_type: ???
+vocoder_name: null
+vocoder_checkpoint_path: null
+
+# The below feature config should match the feature.yaml config used during preprocessing.
+sample_rate: 22050
+win_length: 1024
+hop_length: 256
+
+mel_feature:
+  _target_: nemo.collections.tts.parts.preprocessing.features.MelSpectrogramFeaturizer
+  sample_rate: ${sample_rate}
+  win_length: ${win_length}
+  hop_length: ${hop_length}
+  mel_dim: 80
+  lowfreq: 0
+  highfreq: 8000
```

Comment:
It was discussed before, and I remembered we decided to go with `null` rather than 8000. Please correct me if I am wrong.

Reply:
If we did want to switch it to null, now might be the last good opportunity. Though it may not matter since most experimentation now focuses on 44.1kHz and we will likely add support for training on acoustic tokens/codes soon.

The interesting thing about having highfreq=8000 is it means FastPitch is effectively predicting 16kHz audio while HiFi-GAN upsamples it to 22.05kHz. This allows training the acoustic model on 16kHz data, which greatly increases the amount of training data available.

Reply:
that's true for 16kHz data, but TTS typically recommend a sampling rate at least 24kHz, so I guess directly extracting features from 16kHz data is uncommon in this context.

In any case, when playing with 22050kHz data, nyquist freq (i.e. `null`) may be a standard trick for signal reconstruction purpose? 

I am good with either actions though. @subhankar-ghosh @redoctopus any comments?

Reply:
Fine with changing it to `null`. Out of curiosity, has anyone trained a 22.05kHz model with this set to `null` yet?

Reply:
Changed to `null`.

Reply:
> Fine with changing it to `null`. Out of curiosity, has anyone trained a 22.05kHz model with this set to `null` yet?

Yes, I did for German: https://github.com/NVIDIA/NeMo/blob/main/examples/tts/conf/de/fastpitch_align_22050_grapheme.yaml#L28

Reply:
Great news, I'm glad it was finally changed to `null`, just for consistency purposes.

> The interesting thing about having highfreq=8000 is it means FastPitch is effectively predicting 16kHz audio while HiFi-GAN upsamples it to 22.05kHz. This allows training the acoustic model on 16kHz data, which greatly increases the amount of training data available

The issue with this approach is you need to match STFT window and hop. E.g. 1024 window for 16 kHz becomes 1411.2 for 22.05 kHz. Or, 1024 for 22.05 kHz would turns into 743 for 16 kHz. Tricky to think about (and to set up).

Reply:
> The issue with this approach is you need to match STFT window and hop. E.g. 1024 window for 16 kHz becomes 1411.2 for 22.05 kHz. Or, 1024 for 22.05 kHz would turns into 743 for 16 kHz. Tricky to think about (and to set up).

It is not space efficient, but couldn't you upsample the 16kHz to 22.05kHz and then it would line up?

Reply:
Yes, sure, that is an viable option if you need this setup